### PR TITLE
pptp-1402: redirect to home page if previous question have not been a…

### PR DESCRIPTION
**Pull request by mauro-fanton**
…nswered

### Description of Work carried through

Brief description of what/why/how.

### CODE REVIEW

File: app/controllers/helpers/NonExportedAmountHelper.scala [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import play.api.mvc.Results.Redirect
+
+object NonExportedAmountHelper {
+
+  private def getAmount
+  (
+    page: QuestionPage[Boolean],
+    weightPage: QuestionPage[Long])
+  (implicit request: DataRequest[_]): Option[Long] = {
+    request.userAnswers.get(page).fold[Option[Long]]
```
Comment (liampan) [inline app/controllers/helpers/NonExportedAmountHelper.scala:32]: this is a flatMap :)

File: test/controllers/returns/NonExportedRecycledPlasticPackagingControllerSpec.scala [hunk 2]
```diff
@@ -43,14 +44,20 @@ class NonExportedRecycledPlasticPackagingControllerSpec extends SpecBase with Mo
   val validAnswer = 0L
   //todo: CARL fix flaky tests
```
Comment (liampan) [inline test/controllers/returns/NonExportedRecycledPlasticPackagingControllerSpec.scala:45]: lol

Comment (liampan) [reply test/controllers/returns/NonExportedRecycledPlasticPackagingControllerSpec.scala:45]: @C-Thomp

File: app/controllers/helpers/NonExportedAmountHelper.scala [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+  def nonExportedAmount(implicit request: DataRequest[_]):Either[Result,Long] = {
+
+    (for {
+      manufacturing <- NonExportedAmountHelper.manufacturingPlasticAmount
+      imported <- NonExportedAmountHelper.importedPlasticAmount
+      exported <- NonExportedAmountHelper.exportedAmount
+    } yield manufacturing + imported - exported)
+      .fold[Either[Result, Long]](
+        Left(Redirect(controllers.routes.IndexController.onPageLoad)))(Right(_))
```
Comment (liampan) [inline app/controllers/helpers/NonExportedAmountHelper.scala:54]: not sure about this route living here, but leave for now and we can debate it later :)

**Review by liampan — APPROVED** [review]
tiny comments otherwise all looks good

**Review by C-Thomp — APPROVED** [review]
lgtm

**Pull request merged**